src/react-components/room/MoreMenuPopover.js

```diff
@@ -19,12 +27,12 @@ function MoreMenuItem({ item, closePopover }) {
           target={item.target || "_blank"}
           rel="noopener noreferrer"
         >
-          <Icon />
+          {item.icon?.src ? <img src={item.icon.src} alt={imageAltText} /> : <Icon />}
```

Comment:
can you move this logic to line 11? Then you could leave these as <Icon/> and on line 11 it gets assigned the right asset?